inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -223,25 +229,34 @@ def simulate(
         simulator: simulation.Simulator = stellarators.simulators.Simsopt(),
         resource_pool_id: Optional[UUID] = None,
         run_async: bool = False,
-        plasma_surface_filename: str = 'input.QA',
+        plasma_surface_filepath: Optional[types.Path] = None,
```

Comment:
Do typing.Optional here and on top. Same with uuid.UUID and simplify the imports on top!